packages/core/contracts/common/implementation/FixedPoint.sol

```diff
@@ -390,6 +390,7 @@ library FixedPoint {
 
     function fromSigned(Signed memory a) internal pure returns (Unsigned memory) {
         require(a.rawValue >= 0, "Negative value provided");
+        return Unsigned(uint256(a.rawValue));
```

Comment:
What if `a` is `int_min=-57896044618658097711785492504343953926634992332820282019728792003956564819968`?

Reply:
It would be negative, right? So the require above would catch that I think. Will add a test for that case specifically, though.

Reply:
Done.